Documentation/technical/commit-graph-format.txt

```diff
@@ -17,6 +17,9 @@ metadata, including:
 - The parents of the commit, stored using positional references within
   the graph file.
 
+- The bloom filter of the commit carrying the paths that were changed between
```

Comment:
Commit message: add a question for advice: 

Should we use more generic names here, with respect to the `--changed-paths` option? Perhaps the "Bloom Filter" strings here should be replaced with "Change Paths". Perhaps the version value in the data chunk can be versioned to change the data structure, not just the hash algorithm?